eox_tagging/edxapp_accessors.py

```diff
@@ -4,14 +4,16 @@
 from django.contrib.auth.models import User
 from django.contrib.sites.models import Site
 from django.core.exceptions import ValidationError
-from eox_core.edxapp_wrapper.enrollments import get_enrollment
-from eox_core.edxapp_wrapper.users import get_edxapp_user
 from opaque_keys.edx.keys import CourseKey
 
-try:
-    from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
-except ImportError:
-    from eox_tagging.test.test_utils import CourseOverview  # pylint: disable=ungrouped-imports, useless-suppression
+from eox_tagging.edxapp_wrappers.backends.course_overview_i_v1 import get_course_overview
+from eox_tagging.edxapp_wrappers.backends.enrollment_i_v1 import get_enrollment_dictionary, get_enrollment_object
+from eox_tagging.edxapp_wrappers.backends.users_i_v1 import get_platform_user
+
+CourseEnrollment = get_enrollment_object()
```

Comment:
we can add this to the wrapper file in order to just import and not call the method

Reply:
Yes! That would be better